torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -213,16 +213,16 @@ def from_float(
             group_size = max(block_size)
             compute_dtype = zero_point_dtype if (zero_point_dtype is not None) else input_float.dtype
             device = input_float.device
-            int_data, scale, zero_point, _ = quantize_affine_hqq(input_float, nbits=nbits, group_size=group_size, axis=axis, compute_dtype=compute_dtype, device=device, verbose=False, raw_output=False)
```

Comment:
Renamed int_data to data, as this method is also being used by floatx internally.